Indriaty_Tutut/js/index.js

```diff
@@ -0,0 +1,90 @@
+import { SlideDeck } from './slidedeck.js';
+
+const map = L.map('map', {scrollWheelZoom: false}).setView([0, 0], 0);
+
+// ## The Base Tile Layer
+const mapboxStyle = 'mapbox/streets-v12';
+const mapboxKey = 'pk.eyJ1IjoidHV0dXRpbmQiLCJhIjoiY20weThlMHN1MDE5ZTJtcHZ6NWZ3cHZ0OSJ9.Lkd2ywOlbefcS46ePd5tuA';
+
+const baseLayer = L.tileLayer(
+    `https://api.mapbox.com/styles/v1/${mapboxStyle}/tiles/{z}/{x}/{y}{r}?access_token=${mapboxKey}`,
+    { zoomOffset: -1, tileSize: 512 },
```

Comment:
**Suggestion:** Proper citation of your base map tile is necessary here. Usually map tile providers have their preferred way of citation listed on their websites. Looks like you used a [mapbox tile](https://docs.mapbox.com/help/glossary/attribution/#:~:text=Mapbox%20attribution%20is%20the%20combination,Mapbox%22%20and%20the%20Mapbox%20logo.) that is generated from your mapbox account. In this way, the citation/attribution can be:

```suggestion
{ zoomOffset: -1, 
    tileSize: 512,
    attribution: `© <a href="https://www.mapbox.com/about/maps/">Mapbox</a> © <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> <strong><a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a></strong>`,
},
```